docs/geovation-community-map/community_map_functions.js

```diff
@@ -74,6 +74,89 @@ function toggleInput(id, map) {
     });
 };
 
+function countUniqueCounties(data){
```

Comment:
JS doc for data. What is it ?

Reply:
I call the function in line 177, with recipients being passed in (which is fetched from recipients.json, in line 175)